lalrpop/src/normalize/prevalidate/mod.rs

```diff
@@ -50,6 +50,26 @@ impl<'grammar> Validator<'grammar> {
         for item in &self.grammar.items {
             match *item {
                 GrammarItem::Use(..) => { }
+
+                GrammarItem::MatchToken(ref data) => {
+                    // Ensure that the catch all is final item of final block
+                    let mut contents_idx = 0;
+                    for match_contents in &data.contents {
+                        let mut item_idx = 0;
+                        for item in &match_contents.items {
```

Comment:
I tried that before and it was giving me some obscure message about traits. I may try again.